### DISCUSSION TITLE
Ingest source code and xml

### DISCUSSION THREAD

**Discussion by rollsch** [imartinez/privateGPT#607] [Q&A]
In its current state could I use private GPT to ingest source code and xml files, then ask it questions about the code? Or does it expect natural language?

Im currently testing it out by renaming all my source to .txt and seeing what happens. Curious if anyone else has done this or there are other projects more suited to this?

**Comment by elsaversailles** [comment]
Did changing to .txt extension work? Or the response yield some unnecessary response such as xml based syntaxation?
